fuse_optimizers/src/fixed_lag_smoother.cpp

```diff
@@ -460,6 +470,48 @@ bool FixedLagSmoother::resetServiceCallback(std_srvs::Empty::Request&, std_srvs:
   return true;
 }
 
+bool FixedLagSmoother::stopServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
```

Comment:
To avoid code duplication, can you update the "reset" callback make use of the stop and start callbacks somehow? For example, you could move the start and stop code into "simple" functions. And then have the service callbacks use the simple functions. Something like:
```
stopServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
{
  stop();
}
startServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
{
  start();
}
resetServiceCallback(std_srvs::Empty::Request&, std_srvs::Empty::Response&)
{
  stop();
  start();
}

protected:
void start()
{
  ...
}

void stop()
{
  ...
}
```

Reply:
Done and done